### PR TITLE
README: document simplest usage pattern

### LINKED ISSUE #2622: New option to run on individual files
I'd like to be able to run codespell on one or more specific files, rather than an entire directory. This would essentially have the opposite effect of `-S`/`--skip`.

The option should support filenames, directories, and blob patterns, all three as both a single argument and comma-separated lists. For example, if the option name is `-f`/`--files`:

```
codespell -f foobar.txt
```

```
codespell --files="foo.txt,bar.txt"
```

```
codespell --files="./some/folder"
```

```
codespell --files="*.md,*.txt"
```

### PR DESCRIPTION
**Pull request by waldyrious**
Closes #2622 (/cc @DimitriPapadopoulos).

### CODE REVIEW

**Comment by waldyrious** [comment]
Note: I also took the opportunity to indent any text that "belongs" to code block above it, otherwise the structure of the rendered file could be ambiguous.

File: README.rst [hunk 1]
```diff
@@ -34,43 +34,66 @@ You can use ``pip`` to install codespell with e.g.:
...
 Some noteworthy flags:
 
 .. code-block:: sh
 
     codespell -w, --write-changes
 
-The ``-w`` flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is the same as with doing a dry run. It is recommended to run this with the ``-i`` or ``--interactive`` flag.
+..
+
+    The ``-w`` flag will actually implement the changes recommended by codespell. Not running with ``-w`` flag is the same as with doing a dry run. It is recommended to run this with the ``-i`` or ``--interactive`` flag.
```
Comment (larsoner) [inline README.rst:60]:
```suggestion
The ``-w`` flag will actually implement the changes recommended by codespell. Running without the ``-w`` flag is the same as with doing a dry run. It is recommended to run this with the ``-i`` or ``--interactive`` flag.
```

File: README.rst [hunk 2]
```diff
@@ -34,43 +34,66 @@ You can use ``pip`` to install codespell with e.g.:
...
 
 .. code-block:: sh
 
     codespell -I FILE, --ignore-words=FILE
 
-The ``-I`` flag can be used for a list of certain words to allow that are in the codespell dictionaries. The format of the file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said list of allowed words. **Important note:** The list passed to ``-I`` is case-sensitive based on how it is listed in the codespell dictionaries.
+..
+
+    The ``-I`` flag can be used for a list of certain words to allow that are in the codespell dictionaries. The format of the file is one word per line. Invoke using: ``codespell -I path/to/file.txt`` to execute codespell referencing said list of allowed words. **Important note:** The list passed to ``-I`` is case-sensitive based on how it is listed in the codespell dictionaries.
```
Comment (larsoner) [inline README.rst:68]: These new added indentations will make it render incorrectly. You can "view file" to see how these are rendered as block quotes now -- clicking it:

Screenshot 2022-12-12 at 9 24 05 AM

Will show the problem:

Screenshot 2022-12-12 at 9 24 35 AM

File: README.rst (hunk 1 above)
Comment (waldyrious) [reply README.rst]: Changes applied in c8d5597.

**Review by larsoner — APPROVED** [review]

**Comment by larsoner** [comment]
Enabling automerge, thanks in advance @waldyrious !

**Pull request merged**